src/fuse/ll_request.rs

```diff
@@ -7,7 +7,15 @@ use std::convert::TryFrom;
 use std::ffi::OsStr;
 use std::{error, fmt, mem};
 
-use super::abi::*;
+#[cfg(target_os = "macos")]
+use super::abi::fuse_exchange_in;
+use super::abi::{
```

Comment:
Can not merge?

Reply:
use super::abi::fuse_exchange_in;  This is only needed by macos.